HACKING.md

```diff
@@ -276,3 +276,25 @@ And you can run cockpit-ws and cockpit-bridge under valgrind like this:
 
 Note that cockpit-session and cockpit-bridge will run from the installed
 prefix, rather than your build tree.
+
+# Running Internet Explorer to test Cockpit
+
+While running Firefox or Chrome on your Linux or Mac development machine
+may be easy, some people find it harder to test Internet Explorer. To
+use the following method you need access to the ```windows-8``` testing
+image. This image cannot be freely distributed for licensing reasons.
```

Comment:
We could add a note here that everyone can build their own via `vm-create`, since the base image is freely distributed by Microsoft.